configure

```diff
@@ -100,6 +101,40 @@ while [ "$TF_NEED_HDFS" == "" ]; do
   esac
 done
 
+while [ "$TF_NEED_LIBJPEG_TURBO" == "" ]; do
```

Comment:
Is there any reason why we can't just always use libjpeg-turbo and get rid of the other one? Then we don't need more configure script technical debt.